packages/manager/apps/vrack-services/src/pages/overview/OverviewTab.page.tsx

```diff
@@ -13,7 +13,8 @@ import { DisplayName } from '@/components/display-name/DisplayName.component';
 import { ProductStatusChip } from '@/components/ProductStatusChip.component';
 
 export default function OverviewTab() {
-  const { t, i18n } = useTranslation('vrack-services/dashboard');
+  const { t: tDashboard, i18n } = useTranslation('vrack-services/dashboard');
+  const { t: tCreate } = useTranslation('vrack-services/create');
```

Comment:
**question:** Why creation related translations must be loaded for the dashboard page? If there's common content, you can create a new namespace and just load the shared content.